kagenti/installer/app/components/keycloak.py

```diff
@@ -171,7 +172,23 @@ def install():
         ],
         "Deploying Keycloak with Postgres DB",
     )
-    
+
+    # Configure Keycloak to start with dev mode
+    run_command(
+        [
+            "kubectl",
+            "patch",
+            "statefulset",
+            "keycloak",
+            "-n",
+            "keycloak",
+            "--type=json",
+            "-p",
+            '[{"op": "replace", "path": "/spec/template/spec/containers/0/args", "value":["start-dev"]}]',
```

Comment:
after some testing, I learned we don't even need the `start-dev` - start is fine, so I think we can remove this